### PR TITLE
Simple xeno antagonist

### PR DESCRIPTION
**Pull request by AnturK**
No gameplay changes besides xenomorphs will now be listed in roundend report and check antagonists.
No multiple team support because that would need some rework with origin tracking between stages.

### CODE REVIEW

File: code/modules/antagonists/xeno/xeno.dm [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/datum/team/xeno
+	name = "Aliens"
+
+//Simply lists them.
+/datum/team/xeno/roundend_report()
+	var/list/parts = list()
+	parts += "<span class='header'>The aliens were:</span>"
```
Comment (vuonojenmustaturska) [inline code/modules/antagonists/xeno/xeno.dm]: `The [name] were`, no?

File: code/modules/antagonists/xeno/xeno.dm [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	if(!new_team)
+		for(var/datum/antagonist/xeno/X in GLOB.antagonists)
+			if(!X.owner || !X.xeno_team)
+				continue
+			xeno_team = X.xeno_team
+			return
+		xeno_team = new
+	else
+		if(!istype(new_team))
+			CRASH("Wrong xeno team type provided to create_team")
```
Comment (vuonojenmustaturska) [inline code/modules/antagonists/xeno/xeno.dm]: What if some madman adds xeno ERT, or jolly xenomorph santa?

Comment (AnturK) [reply code/modules/antagonists/xeno/xeno.dm]: This is problem with current team implementation in general, would have to think a bit how to make these less strict.

File: code/modules/antagonists/xeno/xeno.dm (hunk 1 above)
Comment (AnturK) [reply code/modules/antagonists/xeno/xeno.dm]: Yep

**Pull request merged**